### DISCUSSION TITLE
MinerX.3 1.13.0 General Testing and Call Outs

### DISCUSSION THREAD

**Discussion by jennijuju** [filecoin-project/lotus#7463] [Show and tell]
You know the drills 😉 Update your system to lotus [v1.13.0-rc2](https://github.com/filecoin-project/lotus/releases/tag/v1.13.0-rc2) and [start testing](https://github.com/filecoin-project/lotus/discussions/7461#discussioncomment-1439902).

❤️ This post if you are participating in this testing round. If you aren't able to join for reasons, please reach out to @Deep Kapur and me on slack!

## Regular Operations 

Your daily SP operations should stay ✅ .

**Reply with:**
🦭 If you have successfully sealed some new sectors.
🤝 If you have successfully made some deals.
💻 If you passed your first WindowPoSt.
🍷 If you won a new block.

❗ Create [an issue](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fbug&template=bug_report.yml) if you think there is a bug, don't forget to attach the logs!
   - Save the daemon logs, miner logs, and `lotus-miner info all` if you want to report a bug! If the log files are large, you can upload them in `MinerX.3 - v1.13.0 Testing Logs` folder and link it in your issue report/slack message/..

## New feature and Improvements 

Except for [proof v10](https://github.com/filecoin-project/lotus/discussions/7443) and deal-making features and improvements, you should also watch out for:
- Set `LOTUS_PANIC_REPORT_PATH ` and `LOTUS_PANIC_JOURNAL_LOOKBACK ` to get reports generated when a panic occurs on your daemon miner or workers.
- `lotus-miner sectors list` now has an option of `--unproven` to only show sectors that are not in the `Proving` state.
- `lotus-miner storage-deals list` now outputs the storage deals with their respective transfers if you run it with `--format=json`. This makes deal info inspections easier and more readable, you may run `LOTUS_MARKETS_PATH=~/markets-repo-location ./lotus-miner storage-deals --format=json | grep [Proposal CID]` to get information for a specific deal.
- You can now find supported env vars in [default-lotus-miner-config.toml](https://github.com/filecoin-project/lotus/blob/master/documentation/en/default-lotus-miner-config.toml).
- You can see more logs from the actor that records the burning events (what causes the burn and fund movements for the penalty after the 🍫 upgrade.

**Comment by rjan90** [comment]
Is the `PanicReportingPath` and `PanicReportJournalTail` env-variables that should be set, or should they be set in the config.toml?

**Reply by jennijuju** [reply]
Sorry - I think its actually `LOTUS_PANIC_REPORT_PATH` and `LOTUS_PANIC_JOURNAL_LOOKBACK` as env vars

**Reply by rjan90** [reply]
I see! Thanks for the clarification :smile:

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment]
wdPost works, winPost works, sealing works

**Reply by rjan90** [reply]
I can also confirm that the new commands:

- `lotus-miner sectors list --unproven`, is working properly ✔️
- `lotus-miner storage-deals list --format=json`, is working properly ✔️

I have not had any panics on this branch, so I have not gotten to see if the env paths for those reports works yet.

**Comment by kenshyx** [comment]
🦭  🤝   💻

**Comment by stuberman** [comment]
🦭 + 🤝 + 💻 

Still waiting for🍷
